code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -15,15 +17,27 @@
 							PRUDENCE_ATTRIBUTE = 60
 							)
 
+/obj/item/ego_weapon/lamp/attack(mob/living/M, mob/living/user)
+	..()
+	for(var/mob/living/L in livinginrange(1, M))
+		force = 25
```

Comment:
Why are you using force variable? Just make a new one specifically for AOE attack just so we don't have any issues later.
Additionally, you are doing the calculations for EVERY mob in range, instead of only once. Move it all up.